lib/qbo_api/connection.rb

```diff
@@ -68,11 +73,7 @@ def response(resp, entity: nil)
     end
 
     def parse_response_body(resp)
-      body = resp.body
-      case resp.headers['Content-Type']
-      when /json/ then JSON.parse(body)
-      else body
-      end
+      resp.body
```

Comment:
Worth a comment that it's parsed by the middleware now and why this code changed

Reply:
Think it's worth dumping this method completely?

Reply:
Sure, go for it.